Source/JavaScriptCore/runtime/JSGlobalObjectInlines.h

```diff
@@ -28,7 +28,6 @@
 #include "ArrayAllocationProfile.h"
 #include "ArrayConstructor.h"
 #include "ArrayPrototype.h"
-#include "ErrorType.h"
```

Comment:
While the inclusion makes sense the original WebKit source omits this so I am too (closer to original source).